ion_schema_2_0/constraints/annotations-simplified.isl

```diff
@@ -0,0 +1,132 @@
+$ion_schema_2_0
+
+type::{
+  name: 'annotations: closed::[foo, bar]',
+  annotations: closed::[foo, bar],
+}
+
+$test::{
+  type: 'annotations: closed::[foo, bar]',
+  should_accept_as_valid: [
+    0,
+    foo::1,
+    bar::2.0,
+    foo::bar::3e0,
+    foo::foo::'4',
+    bar::foo::bar::foo::"5",
+    foo::null,
+  ],
+  should_reject_as_invalid: [
+    baz::1,
+    foo::baz::2,
+    bar::baz::3,
+    foo::bar::baz::4,
+    document::(),
```

Comment:
(Question) Does this mean we won't be able to use `document` when we have specified `closed annotations`? 
Shouldn't `document` be always allowed as its a built-in type? Or do you think it would be worth mentioning this in the Specification? I saw the following line in the spec but nothing that explicitly say that document will not be allowed when we use a `closed annotation`.

> Applicable to all types except document (since documents cannot be annotated).

Reply:
Thanks for the detailed explanation. It was helpful!